typescript/src/merchi.ts

```diff
@@ -229,12 +229,12 @@ export class Merchi {
     return apiFetch(resource, options);
   };
 
-  public getCurrentUser = () => {
+  public getCurrentUser = (embed?: any) => {
```

Comment:
embed must be an embed descriptor, not `any`.

Reply:
yep.